test/api-test.js

```diff
@@ -50,6 +50,33 @@ describe('RippleAPI', function() {
       _.partial(checkResult, responses.preparePayment, done));
   });
 
+  it('preparePayment - 2', function(done) {
+    this.api.preparePayment(address, requests.preparePayment2, instructions,
+      _.partial(checkResult, responses.preparePayment2, done));
+  });
+
+  it('preparePayment - 3', function(done) {
+    const paymentRequest = _.cloneDeep(requests.preparePayment2);
+    paymentRequest.source.amount.currency = 'USD';
+    paymentRequest.destination.amount.currency = 'LTC';
+    const expected = _.cloneDeep(responses.preparePayment2);
```

Comment:
It's nice to be able to look at the fixture file and directly be able to see exactly what's being tested too.